### PR TITLE
🔧 Summary Card: Change the font size implementation to be simpler & fix import dashboard

### PR DESCRIPTION
**Pull request by MikesGlitch**
I've simplified the font sizing - it should have quicker performance and it's easier to understand. There's a slight spacing adjustment too - It should look less tight but still fill out the space. 

Will need a thorough check to ensure It has the same output depending on the value.

**This also fixes the Import dashboard** when a summary card is being imported. _summary-card_ has been added into the importable report types.

### CODE REVIEW

**Comment by MikesGlitch** [comment]
@lelemm I simplified the font sizing a little bit, can you check and see if I broke anything?

**Comment by lelemm** [comment]
LGTM @MikesGlitch 
The idea of using the height as it is perfect
I'm just curious how did you get the 1.6 factor for the width. lol

**Comment by lelemm** [comment]
> > LGTM @MikesGlitch The idea of using the height as it is perfect I'm just curious how did you get the 1.6 factor for the width. lol
> 
> It just _felt_ right ya know? 😏
🤣👌

> 
> I just fixed a small issue where the suffix wasn't being accounted for in the text length - I think that's everything.
Reviewers must be maintainers or can I approve this for you?

**Comment by MikesGlitch** [comment]
> Reviewers must be maintainers or can I approve this for you?

Anyone can review, but I'll still need a maintainer to approve before merging. Thanks for checking - I wanted to be sure I wasn't breaking anything.

**Comment by lelemm** [comment]
> > Reviewers must be maintainers or can I approve this for you?
> 
> Anyone can review, but I'll still need a maintainer to approve before merging. Thanks for checking - I wanted to be sure I wasn't breaking anything 😅 .

looks good. I wasn't satisfied with the offscreen div neither

**Review by joel-jeremy — APPROVED** [review]

**Pull request merged**
